python/osrd_schemas/osrd_schemas/generated.py

```diff
@@ -98,11 +98,6 @@ class OverlappingSpeedSections(InfraWarningTrait):
     reference: ObjectReference
 
 
-class OverlappingTrackLinks(InfraWarningTrait):
```

Comment:
The version of both python modules should be updated (otherwise it may not update in local installations)